tests/integration/targets/version_sort/runme.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
If you just delete this file and `runme.yml`, it should also work fine - that would be how `filter_dict_kv` and others are doing it. (The `jc` filter needs this extra hoops because it installs a package - since filters are loaded early before tasks are executed, the `jc` filter won't find its requirements on load-time and won't work if the package is installed in the same role as the tests are run in.)

Reply:
Did so and ansible-test worked.